examples/IF_and_HH_responses_to_injection.py

```diff
@@ -0,0 +1,119 @@
+# encoding: utf-8
+"""
+Compare the response of IF and HH neurones when they recieve defferent type of current injections
```

Comment:
"Compare the responses of IF and HH neurons when they receive different types of current injection"